api/lib/domain/usecases/reconcile-oidc-user.js

```diff
@@ -0,0 +1,41 @@
+const { AuthenticationKeyExpired, MissingUserAccountError } = require('../errors');
+const AuthenticationMethod = require('../models/AuthenticationMethod');
+
+module.exports = async function reconcileOidcUser({
+  authenticationKey,
+  oidcAuthenticationService,
+  authenticationSessionService,
+  authenticationMethodRepository,
+  userRepository,
+}) {
+  const sessionContentAndUserInfo = await authenticationSessionService.getByKey(authenticationKey);
+  if (!sessionContentAndUserInfo) {
+    throw new AuthenticationKeyExpired();
+  }
+
+  const { userInfo, sessionContent } = sessionContentAndUserInfo;
+  if (!userInfo?.userId) {
+    throw new MissingUserAccountError();
+  }
+
+  const { userId, externalIdentityId } = userInfo;
```

Comment:
Comme une impression ... 
![image](https://user-images.githubusercontent.com/38167520/187883301-bf0292f4-0ebe-4fa2-a62d-985c1b9c750b.png)

Reply:
je comprends pas 🙈 😂

Reply:
parce qu'on déstructure d'une déstructuration , ça fait un peu poupées russes :P

Reply:
hahahahahaha

Reply:
j'aurai pu le faire sur la meme ligne mais je ne pense pas que ca vous plairait 😉 genre comme ca

```suggestion
  const { userInfo: { userId, externalIdentityId} = {}, sessionContent } = sessionContentAndUserInfo;
  if (!userId) {
    throw new MissingUserAccountError();
  }
```

Reply:
Non je préfère comme tu l'as écris dans le code, c'est plus lisible :P